### ISSUE TITLE
submodule/src/example.cpp: errors in identify_pc

### ISSUE THREAD

**Issue by shekmun** [open-license-manager/examples#11]
In the example file [https://github.com/open-license-manager/examples/blob/develop/submodule/src/example.cpp](url)
```size_t pc_id_sz = LCC_API_PC_IDENTIFIER_SIZE;```
should be modified to
```size_t pc_id_sz = LCC_API_PC_IDENTIFIER_SIZE + 1;```
or ```identify_pc``` will fail and print ```errors in identify_pc```. It won't output the pc signature.

**Comment by gcontini** [comment]
yes... it was correct when you reported it. I changed the logic now, LCC_API_PC_IDENTIFIER_SIZE contains the space for the ending '\0' . 

Anyway it's fixed.
